tutorials/tts/Fastpitch_Training_GermanTTS.ipynb

```diff
@@ -0,0 +1,1112 @@
+    "manifest_filepath: \"train_manifest.json\"\n",
+    "sup_data_path: \"sup_data\"\n",
+    "sup_data_types: [ \"align_prior_matrix\", \"pitch\" ]\n",
+    "whitelist_path: \"nemo_text_processing/text_normalization/de/data/whitelist.tsv\"\n",
+    "\n",
+    "dataset:\n",
+    "  _target_: nemo.collections.tts.torch.data.TTSDataset\n",
+    "  manifest_filepath: ${manifest_filepath}\n",
+    "  sample_rate: 22050\n",
+    "  sup_data_path: ${sup_data_path}\n",
+    "  sup_data_types: ${sup_data_types}\n",
+    "  n_fft: 1024\n",
+    "  win_length: 1024\n",
+    "  hop_length: 256\n",
+    "  window: \"hann\"\n",
+    "  n_mels: 80\n",
+    "  lowfreq: 0\n",
+    "  highfreq: 8000\n",
+    "  max_duration: null\n",
+    "  min_duration: 0.1\n",
+    "  ignore_file: null\n",
+    "  trim: false\n",
+    "  pitch_fmin: 65.40639132514966\n",
+    "  pitch_fmax: 2093.004522404789\n",
+    "\n",
+    "  text_normalizer:\n",
+    "    _target_: nemo_text_processing.text_normalization.normalize.Normalizer\n",
+    "    lang: de\n",
+    "    input_case: cased\n",
+    "    whitelist: ${whitelist_path}\n",
+    "\n",
+    "  text_normalizer_call_kwargs:\n",
+    "    verbose: false\n",
+    "    punct_pre_process: true\n",
+    "    punct_post_process: true\n",
+    "\n",
+    "  text_tokenizer:\n",
+    "    _target_: nemo.collections.tts.torch.tts_tokenizers.GermanCharsTokenizer\n",
+    "    punct: true\n",
+    "    apostrophe: true\n",
+    "    pad_with_space: true\n",
+    "    phonemes: true\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1ba566f2",
+   "metadata": {},
+   "source": [
+    "## 5. Create Supplementary Data\n",
+    "\n",
+    "As mentioned in the [FastPitch_MixerTTS_Training.ipynb](FastPitch_MixerTTS_Training.ipynb) - To accelerate and stabilize our training, we also need to extract pitch for every audio, estimate pitch statistics (mean and std) and pre-calculate alignment prior matrices for alignment framework. To do this, all we need to do is iterate over our data one time, via `extract_sup_data.py` script.\n",
+    "\n",
+    "\n",
+    "Example:\n",
+    "```bash\n",
+    "python scripts/dataset_processing/tts/extract_sup_data.py --config-path openslr/ds_conf --config-name ds_for_fastpitch_align.yaml manifest_filepath=/Data/thorsten-de/train_phonemes_manifest.json sup_data_path=/Data/thorsten-de/phonemes/\n",
+    "```\n",
+    "\n",
+    "The above example gives the following result\n",
+    "1. Creates two folders under `sup_data_path` - `pitch` and `align_prior_matrix`\n",
+    "2. prints out `PITCH_MEAN, PITCH_STD = 132.524658203125, 37.389366149902344`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8ff9c449",
+   "metadata": {},
+   "source": [
+    "# Training"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "36dea1f4",
+   "metadata": {},
+   "source": [
+    "Before we train our model, let's define model config:\n",
+    "\n",
+    "```yaml\n",
+    "name: FastPitch\n",
```

Comment:
Please cut the following down to only the config lines that are changed from the default config (plus relevant context). Having the whole config in here makes it difficult to know what to focus on--consider linking the file for the people who want to see the whole thing.